lib/AvaTaxClient.js

```diff
@@ -74,7 +75,7 @@ export default class AvaTaxClient {
    * @param   string  payload    The request body, if this is being sent to a POST/PUT API call
    */
   restCall({ url, verb, payload }) {
-    return fetch(url, {
+    return withTimeout(1200000, fetch(url, {
       method: verb,
```

Comment:
Have you looked into AbortController? Might be more appropriate for canceling a fetch

Reply:
The focus is adjusting the timeout rate to match what's in REST, not so much canceling fetch on timeout